level.js

```diff
@@ -1,8 +1,8 @@
 function Level(font) {
+	this.ancient_technology = ["VCR", "Cassette", "CD", "Delorean", "CRT", "FTP", "CVS", "BetaMax", "Polaroid", "Ceefax", "PDA", "Pager", "Walkman", "Discman"]
```

Comment:
Yes sure.
I have no problem with that, hahaha.

On Sat, Aug 27, 2016 at 7:57 PM, Lee Reilly <notifications@github.com>
wrote:

> In level.js
> <https://github.com/nickidebruyn/p5games/pull/1#discussion_r76519941>:
>
> > @@ -1,8 +1,8 @@
> >  function Level(font) {
> > +	this.ancient_technology = ["VCR", "Cassette", "CD", "Delorean", "CRT", "FTP", "CVS", "BetaMax", "Polaroid", "Ceefax", "PDA", "Pager", "Walkman", "Discman"]
>
> @povrazor <https://github.com/povrazor> - can we add the old Ludum Dare
> servers here? [image: :trollface:]
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nickidebruyn/p5games/pull/1/files/dbbf5f0f0ed2b4fd80dc52b80a0aa46b0d13f3b3#r76519941>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKQ5FKQTg9bwBio-OnB_iETuPm0mRcTqks5qkHqHgaJpZM4JuoK1>
> .
>



-- 
Life is a joy!!!